ccpp/physics_namelists/input_GFS_v17_HR3_RRTMGP.nml

```diff
@@ -0,0 +1,171 @@
+&gfs_physics_nml
```

Comment:
The changes I mentioned below are based on @lisa-bengtsson and Jiande's HR3 configuration. There are also applied to input_GFS_v17_HR3.nml

Reply:
@mzhangw The GP nml is identical to the HR3 SDF, with one exception, do_RRTMGP to True.

Can you open a PR into this branch with the updated input_GFS_v17_HR3.nml from @lisa-bengtsson? And I will update the GP counterpart in here.

Reply:
will do.